go/pkg/bertyaccount/service_grpc.go

```diff
@@ -247,3 +251,22 @@ func getSerivceError(err error) *Error {
 		Message:       err.Error(),
 	}
 }
+
+func (s *service) recv(id string, cstream *grpcutil.LazyStream) (*ClientStreamRecv_Reply, error) {
```

Comment:
can you update the name of this function to be more explicit?
(something like clientStreamRecv/streamRecv should be good)